stress/shell/src/main/java/alluxio/stress/cli/client/StressClientIOBench.java

```diff
@@ -128,9 +140,21 @@ public void prepare() throws Exception {
     for (Map.Entry<String, String> entry : mParameters.mConf.entrySet()) {
       hdfsConf.set(entry.getKey(), entry.getValue());
     }
-    mCachedFs = new FileSystem[mParameters.mClients];
-    for (int i = 0; i < mCachedFs.length; i++) {
-      mCachedFs[i] = FileSystem.get(new URI(mParameters.mBasePath), hdfsConf);
+
+    if (mParameters.mClientType == FileSystemClientType.ALLUXIO_HDFS) {
+      mCachedFs = new FileSystem[mParameters.mClients];
```

Comment:
For better tracing, you can log an INFO message on which FS is used. Not choosing DEBUG here because it's in the user logging and we rarely configure user side log to DEBUG.

Reply:
added info log.